src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -244,9 +244,9 @@ In that case you can also skip the `sentry-cli` configuration step below.
 
 ### Use `hidden-source-map`
 
-_(New in version 6.17.1)_
+_(New in version 6.17.1, will default to `true` in 8.0.0 and beyond.)_
 
-If you would like to use `hidden-source-map` rather than `source-map` as your webpack `devtool`, so that your built files do not contain a `sourceMappingURL` comment, add a `sentry` object to `moduleExports` above, and set the `hideSourceMaps` option to `true`:
+Depending on your deployment setup, adding `sentry/nextjs` to your app may cause your source code to be visible in browser devtools when it wasn't before. (This happens because of the default behavior of [Webpack's `source-map` built-in `devtool`](https://webpack.js.org/configuration/devtool/).) To prevent this, you can use `hidden-source-map` rather than `source-map`, which will prevent your built files from containing a `sourceMappingURL` comment, thus making sourcemaps invisible to the browser. To use `hidden-source-map`, add a `sentry` object to `moduleExports` above, and set the `hideSourceMaps` option to `true`:
```

Comment:
The way you've used the word "above" in that last sentence is a bit confusing. At first I thought I needed to refer to code snippet previous to this one. If you said "above it" instead, would that work?

Reply:
No, because there's no real antecedent for "it" there, but I can reword to make it clearer.